### PR TITLE
More goog removal

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

File: src/ol3cesium.js [hunk 1]
```diff
@@ -59,17 +59,21 @@ olcs.OLCesium = function(options) {
    * @type {!Element}
    * @private
    */
-  this.container_ = goog.dom.createDom(goog.dom.TagName.DIV,
-      {style: fillArea + 'visibility:hidden;'});
+  this.container_ = document.createElement('DIV');
+  var containerAttribute = document.createAttribute('style');
```
Comment (fredj) [inline src/ol3cesium.js]: Is `createAttribute` really needed? simply:
```js
this.container_ = document.createElement('DIV');
this.container_.style = fillArea + 'visibility:hidden;';
```
untested

File: src/ol3cesium.js [hunk 2]
```diff
@@ -84,12 +88,16 @@ olcs.OLCesium = function(options) {
    * @type {!HTMLCanvasElement}
    * @private
    */
-  this.canvas_ = /** @type {!HTMLCanvasElement} */
-      (goog.dom.createDom(goog.dom.TagName.CANVAS, {style: fillArea}));
+  this.canvas_ = /** @type {!HTMLCanvasElement} */ (
+      document.createElement('CANVAS'));
+  var canvasAttribute = document.createAttribute('style');
```
Comment (fredj) [inline src/ol3cesium.js]: idem

File: src/ol3cesium.js (hunk 1 above)
Comment (gberaudo) [inline src/ol3cesium.js]: Unfortunately, `element.style` does not accept a string.

**Comment by fredj** [comment]
LGTM

**Comment by gberaudo** [comment]
Thanks for the review @fredj.

**Pull request merged**